pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,14 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    def test_read_zero_chunked_array(self, pa):
+        df = pd.DataFrame({"value": pd.array([], dtype=pd.Int64Dtype())})
```

Comment:
can you assert that this is a zero-chunked array for impl.

Reply:
I'm sorry, but I don't understand what you are trying to point out.
What are your concerns?

Reply:
the title of the issue is misleading actually. nvm

Reply:
Would it be better to change the title from `zero-chunked` to `empty-array`?

Reply:
yes